docs/rules/capitalized-comments.md

```diff
@@ -190,6 +190,15 @@ Examples of **correct** code with `ignoreConsecutiveComments` set to `true`:
  */
```

Comment:
Should we also update this example comment to avoid any misunderstanding? (Specifically, the parts that mention a "valid" comment)